docs/operations-guide/running-metabase-on-elastic-beanstalk.md

```diff
@@ -100,31 +143,68 @@ The database settings screen will give you a number of options for your applicat
 
 Once you've entered a password and clicked `Save`, the red outline on the Database box should have gone away, indicating that the application is valid and ready to be launched.
 
-### Additional Options
+#### Configuring RDS for Metabase OUTSIDE ElasticBeanstalk (recommended)
+
+In AWS Console, enter RDS in the search box or select the service from the dropdown button on the top left of the page.
+
+Once inside RDS, click on the `Create database` button
+![RDS Create Database](images/RDSCreateDatabase.png)
+
+On the Create Database menu, you have to select MySQL or PostgreSQL as engine types, as these two are the ones that Metabase support as the Application Database (where Metabase will save all of its configurations). For this example we will choose PostgreSQL on its latest version available in AWS at this time (12.4-R1)
+![RDS Postgres Selection](images/RDSPostgresSelection.png)
```

Comment:
Do we need this screenshot?

As a rule, try to describe, succinctly, what people need to do, and only then think: is that sufficient, or would the addition of a screenshot significantly improve what I need to communicate?

Reply:
wiped a couple of screenshots, you are absolutely right, as this is my first guide I'm being screenshotholic